doc/tutorials/snippet-sharing-rest-service.md

```diff
@@ -0,0 +1,196 @@
+[website](http://cppcms.com/wikipp/en/page/cppcms_1x_build).
+It contains also a [guide](http://cppcms.com/wikipp/en/page/apt)
+explaining the installation through a dependency manager.
+Unfortunately, no repository is available for Debian Jessie yet.
+
+##### Jansson #####
+
+The Jansson library supports working with json data in C.
+To install it, use the following steps:
+
+- Download the (latest) source from [their website](http://www.digip.org/jansson/releases/)
+- Extract the archive: `bunzip2 -c jansson-2.x.tar.bz2 | tar xf - && cd jansson-2.x` (replace 2.x with your version)
+- Configure the build: `mkdir build && cd build && cmake ..`
+- Execute the build, run tests and install: `make && make check && make install`
+
+##### LibJWT #####
+
+Now we can also install LibJWT:
+
+- Download the (latest) source from [GitHub](https://github.com/benmcollins/libjwt/releases)
+- Extract the archive: `tar -xjf v1.x.x && cd libjwt-1.x.x` (replace 1.x.x with your version)
+- Configure the build: `mkdir build && cd build && cmake ..`
+- Execute the build and run tests: `make jwt && make jwt_static && make check`
+
+Alternatively LibJWT can also be installed through a pre-built APT package:
+
+- Add APT repository: `add-apt-repository ppa:ben-collins/libjwt`
+- Install via apt-get: `apt-get update && apt-get install libjwt`
+
+#### Frontend ####
+
+The frontend does only require the package manager [npm](https://www.npmjs.com/).
+It is preferred to install it along with [nodeJS](https://nodejs.org/).
+
+To install npm via APT, use: `apt-get install npm`
+
+### Build the Applications ###
+
+After installing the dependencies, we are ready to build the applications.
+To do so, we can follow the steps explained in the [build guide](/doc/COMPILE.md).
+Make sure to include the two tools `rest-backend` and `rest-frontend`, e.g. by
+using the arguments `-DTOOLS=ALL` or `-DTOOLS=rest-backend;rest-frontend`.
+
+## Configuration ##
+
+The applications configure themselves as much as possible during build,
+but some settings have to be set manually afterwards.
+
+### Backend ###
+
+To make sure that the configuration specification was set properly during installation,
+use the command `kdb mount`. You should see a list of mountpoints, of which one should
+be something like `/sw/elektra/restbackend/#0`. If you do not see this mountpoint,
+use `kdb mount rest-backend-spec.ini /sw/elektra/restbackend/#0 ni` to mount it manually.
+
+After that you need to set three additional configuration parameters that have no defaults.
+It is recommended to set them for the system namespace if you will use a tool like
+`systemctl` to manage the services. (For the api_specification key, see below!)
+```
+kdb set system/sw/elektra/restbackend/#0/current/backend/jwt/encryption_key "use_a_secret_key_here!"
+kdb set system/sw/elektra/restbackend/#0/current/backend/api_specification/html "http://link.to/the/html/version/of/api/description"
```

Comment:
In `doc/api_blueprints/snippet-sharing.apib` you proposed `restapi.libelektra.org` or is this only for the restapi itself? If so, we could use `doc.libelektra.org/restapi/$version` ($version as it is for the main api)?

Will there be any problems if we generate daniels and your parts together?